src/test/resources/testng.xml

```diff
@@ -6,6 +6,7 @@
   <suite-files>
     <suite-file path="./junit-suite.xml" />
     <suite-file path="./parent-module-suite.xml" />
+    <suite-file path="./test/listeners/github1284/github1284.xml" />
```

Comment:
Use a full standalone sample instead of adding it into the global suite. 
And the current test is too simple IMO. I think we should have at least 2 suite files and check if the listeners are called at the good time.